test/Driver/options.swift

```diff
@@ -124,6 +124,13 @@
 // RUN: not %swiftc_driver -dwarf-version=6 %s 2>&1 | %FileCheck -check-prefix INVALID_DWARF_VERSION %s
 // INVALID_DWARF_VERSION: invalid value '{{1|6}}' in '-dwarf-version={{1|6}}'
 
+// RUN: %swift_driver -### -g -dwarf-fission -o %t/dwarf-fission.o %s 2>&1 | %FileCheck -check-prefix DWARF_FISSION %s
```

Comment:
Looks like on Linux we invoke `bin/swift` here instead of `bin/swiftc`. Would I have to pass the `-Xfrontend` prefix before `-dwarf-fission` somehow or add support for it elsewhere? Or is this using https://github.com/apple/swift-dirver here?

The test failure was:
```
/home/build-user/swift/test/Driver/options.swift:128:19: error: DWARF_FISSION: expected string not found in input
// DWARF_FISSION: -split-dwarf-output {{.*}}/dwarf-fission.dwo
                  ^
<stdin>:1:1: note: scanning from here
<unknown>:0: error: option '-o' is not supported by 'swift'; did you mean to use 'swiftc'?
^
<stdin>:1:59: note: possible intended match here
<unknown>:0: error: option '-o' is not supported by 'swift'; did you mean to use 'swiftc'?
                                                          ^
```

Also, interesting that on Windows it's there:
```
> bin\swift --help | grep fission
  -dwarf-fission          Emit debug info as split-DWARF (Fission) in a separate .dwo file
```

Reply:
Not sure, but as I see this above at line 70, `%swift_driver` doesn't take `-o`? Do we need `%target-swift-frontend%` instead?

```
// RUN: not %swift_driver -### -o %t %s 2>&1 | %FileCheck -check-prefix=ARG_o %s
// ARG_o: error: option '-o' is not supported by 'swift'; did you mean to use 'swiftc'?
```

Reply:
Haha, good point thanks! Did this in the evening in too much of a hurry. Let me revisit tomorrow.